cmd/kubeadm/app/constants/constants.go

```diff
@@ -191,9 +191,9 @@ const (
 	// Default behaviour is 24 hours
 	DefaultTokenDuration = 24 * time.Hour
 
-	// LabelNodeRoleMaster specifies that a node is a master
+	// LabelNodeRoleControlPlane specifies that a node is a control-plane
 	// This is a duplicate definition of the constant in pkg/controller/service/service_controller.go
-	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
+	LabelNodeRoleControlPlane = "node-role.kubernetes.io/master"
```

Comment:
Don't change this one. It's a local copy of an external constant, that will eventually be removed and pointed again to the external reference.